### ISSUE TITLE
Actively Maintained?

### ISSUE THREAD

**Issue by tsteuwer** [Endlessages/Information#1]
Hey, just trying to see if this is actively being worked on. I've been getting into Blender and Unity and would like to offer some assistance if needed. I'm a developer by trade, so let me know if there's anything I can do. This has and will also be my favorite game. It was way beyond it's years, even compared to most games now. You had FPS, RPG, Shapeshifting, Magic, Melee, and all sorts of stuff. I wish they would remake this game so badly. I'm totally willing to contribute to this. BTW, I HATE the last expansion they did on this (the floating rocks zones and the remake of the newbie zone lol). I want it to go back to the way it was in Beta/Gold.

**Comment by HelloKitty** [comment]
@tsteuwer Hello friend. For a little while I did do some limited network emulation work on Endless Ages. I considered that a dead end though. Spent a bit of time extracting some of the 3D content instead and building some demo scenes in Unity3D as you can see here: https://www.youtube.com/watch?v=7exgvAE6Mis

However I eventually decided to stop working on the project because there wasn't really a path to creating or delivering something in a reasonable amount of time that anyone would be interested in. I do continue to work in game emulation areas and have been exploring ways to help developers create massive online games in afew ways. Maybe Endless Ages could benefit from that someday, that work is done here: https://github.com/gladmmo/GladMMO.Library and most recently on the TrinityCore3 branch. Creating a general MMORPG backend and client in Unity3D that is compatible with WoW emulation.

Other than that, I have no news to report sadly.

Anyone is of course free to do anything they like when it comes to Endless Ages stuff ^,^. But I don't quite have an idea on something people, such as me and others, could collaborate on right now to make something.

**Comment by tsteuwer** [comment]
@HelloKitty, man, that makes me sad. I just loaded up the client repo and got it running in unity 2019 and was going to submit a PR. I was hoping one day someone would bring this game back to life and I could contribute to making it better looking and adding more content. Where did you get the source from? Is there more? Would live to get the maps and character models and see if I can create some more scenes. Getting it running brought back the biggest nostalgia rush I've had in a very long time.

**Comment by HelloKitty** [comment]
@tsteuwer Sorry to deliver bad news =(. I can't even remember what is in that repository, it's been afew years now. My approach to creating these scenes was to load up the Endless Ages game engine editor and load the target scene and then dump the rendered scene with frame dump tools like 3DXRipper. A lot of cleanup work was required from there. Unfortunately the last time I went to go run the editor it wouldn't launch, and I've forgotten what needs to be done to get it to run.

Here are some zips of the data I have had lying around for awhile. https://drive.google.com/drive/folders/1XPebzlCZ0TunbTyIxjw9cqNkaQ4f7qxT?usp=sharing

**Comment by tsteuwer** [comment]
Ok 😞 thanks for cheering me up for a good 30 mins. Getting to run around the newbie zone made my day.

**Comment by tsteuwer** [comment]
Btw, can you actually run the game and get in it without a backend server to login to?

**Comment by HelloKitty** [comment]
@tsteuwer I don't think so, but then again I don't know. I did not continue reverse engineering things to that point to find out.

**Comment by tsteuwer** [comment]
No worries. I have a copy of the backend services if you're interested. They were open sourced by the last owner.

**Comment by HelloKitty** [comment]
Really @tsteuwer ? I was under the impression nobody has the source code for the original backend. Or are they just compiled binaries?

**Comment by tsteuwer** [comment]
Are you on discord? This would be easier to talk about over that.

**Comment by HelloKitty** [comment]
@tsteuwer Yes, my Discord is HelloKitty#0971

**Comment by tsteuwer-accesso** [comment]
Sent a request. I'm Orani on disc
